build.gradle.kts

```diff
@@ -253,42 +252,3 @@ allprojects {
         }
     }
 }
-
-dependencies {
-    implementation(project(":darklaf-native-utils"))
-    implementation(project(":darklaf-windows"))
-    implementation("com.formdev:svgSalamander")
-    implementation("net.java.dev.jna:jna")
-    implementation("org.swinglabs:jxlayer")
-    compileOnly("org.swinglabs:swingx")
-    testImplementation("org.swinglabs:swingx")
-}
-
-tasks.jar {
-    CrLfSpec(LineEndings.LF).run {
-        into("META-INF") {
-            filteringCharset = "UTF-8"
-            textFrom("licenses/NOTICE.txt")
-            textFrom("licenses/PBJAR_LICENSE.txt")
-            textFrom("licenses/INTELLIJ_LICENSE.txt")
-        }
-    }
-}
-
-tasks.shadowJar {
-    exclude("help/")
-    exclude("icons/")
-    exclude("org/jdesktop/jxlayer/plaf/ext/images/")
-    exclude("com/sun/jna/darwin/")
-    exclude("com/sun/jna/freebsd-x86/")
```

Comment:
If `shadowJar` is not used much, then I can remove it.

Reply:
Yes would have done that anyway :)